### ISSUE TITLE
 Adding tag statistics, analysis and jumping functions

### ISSUE THREAD

**Issue by cliuxinxin** [doccano/doccano#1908]
Feature description
---------

When I use the doccano annotation ner, I often export all the labels to refine for analysis. Is it possible to have a statistics screen where you can see the individual labels and jump to that position for fixing.

**Comment by Hironsan** [comment]
Could you please show me some simple image to make us understand the feature?

**Comment by cliuxinxin** [comment]
OK, this is my anlysis page with refine

![image](https://user-images.githubusercontent.com/6162238/179135399-90fc32da-98c5-4e37-a72d-21f74b739e7d.png)

**Comment by Hironsan** [comment]
Thanks. Is this enough?

![image](https://user-images.githubusercontent.com/6737785/179138285-6a4b1523-33be-41de-ac9f-3c1ae3b5a1e5.png)

**Comment by cliuxinxin** [comment]
yes ,maybe search can filter the label,and when we click the label,it can jump to the label position.
